plugins/core/Standard/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+
+add_subdirectory( ${CMAKE_CURRENT_SOURCE_DIR}/qAnimation )
```

Comment:
do you really need to add ```${CMAKE_CURRENT_SOURCE_DIR}/```?

Reply:
Nope! I'd copy-pasted all of those from the original file - not sure why they had that. Probably my misunderstanding cmake when I did that a couple of years ago.